pkg/manifest/os.go

```diff
@@ -612,6 +612,15 @@ func (p *OS) serialize() (osbuild.Pipeline, error) {
 		pipeline.AddStage(osbuild.GenGroupsStage(p.OSCustomizations.Groups))
 	}
 
+	if p.OSCustomizations.LockRootUser {
+		usersOptions, err := osbuild.NewUsersStageOptionsForLockedRoot()
+		if err != nil {
+			return osbuild.Pipeline{}, err
+		}
+
+		pipeline.AddStage(osbuild.NewUsersStage(usersOptions))
```

Comment:
Without the error return in `osbuild.NewUsersStageOptionsForLockedRoot()` this could be a bit shorter:
```go
	if p.OSCustomizations.LockRootUser {
		pipeline.AddStage(osbuild.NewUsersStage(osbuild.NewUsersStageOptionsForLockedRoot())
   }
```